### PR TITLE
Fix NPM security updates

### PR DESCRIPTION
**Pull request by thekiwi**
Fix: Update all critical vulnerable dependencies (mpath & handlebars)
Fix: Update other high risk vulnerabilities.

Bumped dependencies:

main: mongoose, webpack-dev-server, tar, path-parse, prismjs
examples: swagger-ui-dist
dev: lerna / nyc

### CODE REVIEW

**Comment by juanjoseruiz** [comment]
hey @thekiwi I reviewed the list of packages you upgraded up to this point and I feel comfortable with all of them:

- lerna / nyc are just dev dependencies
- mongoose, I am familiar with this upgrade and should not cause issues (on siteflow we are already using 5.13.9 and newer changes are just fixes). we should also be fine with the latest mongodb driver upgrade.
- swagger-ui-dist is only used on examples
- webpack-dev-server this also contains fixes only

I think we can safely bump a few ones more, will continue with such work and will mark as ready for review once I'm done with it.

**Comment by juanjoseruiz** [comment]
@thekiwi @pierissimo this is the initial set of libraries I am more familiar with. I think we should be fine with all those upgrades. After this, I will check if we can also resolve some others and I will document an exception for the ones that cannot be fixed without significant effort.

File: examples/rest-custom-ajv/package.json [hunk 1]
```diff
@@ -17,8 +17,8 @@
 		"debug": "^4.1.1",
 		"express": "4.16.4",
 		"lodash": "4.17.21",
-		"mongoose": "^5.11.15",
-		"swagger-ui-dist": "^3.35.1"
+		"mongoose": "^5.13.14",
+		"swagger-ui-dist": "^4.1.3"
```
Comment (pierissimo) [inline examples/rest-custom-ajv/package.json:21]: it's worth checking if the swagger ui is working after the upgrade, you can do it by visiting the `/explorer` path of a running example

**Review by pierissimo — APPROVED** [review]

File: examples/rest-custom-ajv/package.json (hunk 1 above)
Comment (juanjoseruiz) [reply examples/rest-custom-ajv/package.json:21]: hey Piero, I tested all the examples and the swagger is displaying correctly. I also tested the APIs a bit and seems to be responding fine.

**Pull request merged**